server/src/projects/project-applicants/project-applicant.controller.ts

```diff
@@ -41,7 +45,7 @@ export class ProjectApplicantController {
     const { records } = await this.crmService.get('contacts', `
       $select=${CONTACT_ATTRS.join(',')}
       &$filter=startswith(emailaddress1, '${email}')
-        and statuscode eq ${ACTIVE_CODE}
+        and statuscode eq ${ACTIVE_STATUSCODE}
```

Comment:
Also another note -- this was removed in another PR that's now merged into develop. We end up grab all contacts regardless of statuscode